tests/imagebasedupgrade/validations/post_ibu_validations.go

```diff
@@ -0,0 +1,28 @@
+package ibuvalidations
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/polarion"
+	"github.com/openshift-kni/eco-gosystem/tests/imagebasedupgrade/internal/imagebasedupgradeparams"
+)
+
+// PostUpgradeValidations is a dedicated func to run post upgrade test validations.
+func PostUpgradeValidations() {
+	Describe(
+		"PostIBUValidations",
+		Label("PostIBUValidations"), func() {
+			It("Validate Cluster version", polarion.ID("99999"), Label("ValidateClusterID"), func() {
```

Comment:
This test case validates "Cluster version" post upgrade . From my view, Label has a typo error - Should be ValidateClusterVersion

Reply:
Thanks, updated.